.github/workflows/pythonpackage_future3.yml

```diff
@@ -6,13 +6,11 @@ on:
         - 'future3/**'
     paths:
         - '**.py'
-        - '**.py.*'
   pull_request:
     branches:
         - 'future3/**'
     paths:
         - '**.py'
-        - '**.py.*'
```

Comment:
The `.flake8` config file still has this file include `filename = *.py,*.py.*` . Should there be removed aswell if not needed.

Reply:
agreed